### PR TITLE
test(op-stack): refine unit tests

### PR DESCRIPTION
**Pull request by dohaki**
Closes ACX-2206

Refines existing unit tests for "old" OP stack adapters.

### CODE REVIEW

File: contracts/MockCctpEvents.sol [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+contract CctpMessageTransmitter {
+    mapping(bytes32 => uint256) public usedNonces;
+
+    function setUsedNonce(bytes32 nonceHash, bool isUsed) external {
+        if (isUsed) {
+            usedNonces[nonceHash] = 1;
+        } else {
+            usedNonces[nonceHash] = 0;
```
Comment (bmzig) [inline contracts/MockCctpEvents.sol:50]:
```suggestion
usedNonces[nonceHash] = isUsed ? 1 : 0;
```

**Comment by bmzig** [comment]
Everything else LGTM.

File: contracts/MockCctpEvents.sol (hunk 1 above)
Comment (dohaki) [reply contracts/MockCctpEvents.sol]: Nice didn't know that Solidity has ternaries by now

**Review by bmzig — APPROVED** [review]

File: src/clients/bridges/op-stack/OpStackAdapter.ts [hunk 2]
```diff
@@ -66,7 +66,7 @@ export class OpStackAdapter extends BaseAdapter {
     // which maps to either the CCTP or OVM Standard bridge depending on the request.
     const usdcAddress = TOKEN_SYMBOLS_MAP.USDC.addresses[this.hubChainId];
     const l2NativeUsdcAddress = TOKEN_SYMBOLS_MAP.USDC.addresses[this.chainId];
-    if (usdcAddress && l2NativeUsdcAddress) {
+    if (usdcAddress && l2NativeUsdcAddress && !this.customBridges[usdcAddress]) {
```
Comment (pxrl) [inline src/clients/bridges/op-stack/OpStackAdapter.ts:69]: I refactored the code above recently, such that `customBridges` is assigned entirely within this constructor, so this is now the only place we actually assign custom USDC. I think it's probably OK to either drop this check, or assert on it on the line above. Either are fine w/ me.

**Review by pxrl — APPROVED** [review]

File: src/clients/bridges/op-stack/OpStackAdapter.ts (hunk 2 above)
Comment (dohaki) [reply src/clients/bridges/op-stack/OpStackAdapter.ts]: yea good point. removed it 👍

**Pull request merged**